donkeycar/parts/camera.py

```diff
@@ -1,10 +1,17 @@
+import logging
 import os
 import time
 import numpy as np
 from PIL import Image
 import glob
 from donkeycar.utils import rgb2gray
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
We are already setting the logging level in the car app (in `complete.py`) to INFO already. Because this goes to the root logger, you would optimally set this only once where you start the application. There is no harm with this line, but if we decide to dynamically change the logger level per argument or config then we have to find and remove all these local settings as they might interfere/overwrite the application settings.